standalone/portal-api/src/modules/files/services/files.service.ts

```diff
@@ -33,6 +33,10 @@ import { FileParticipantRepository } from '../repositories/file-participant.repo
 import { SignatureService } from './onchain/signature.service';
 import { triggerError } from '../utils/trigger.errror';
 
+type ICreateFile = {
```

Comment:
Please move to own file, also name it simply CreateFile, prefixing it with 'I' introduces the confusion that this might be an interface